### PR TITLE
Add a "bufferedamountlow" event.

### PR DESCRIPTION
**Pull request by bemasc**
Fixes https://github.com/w3c/webrtc-pc/issues/232

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -3227,6 +3227,24 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
...
+          "#dom-datachannel-bufferedamount">bufferedAmount</a></code> is
+          considered to be low.  When the <code><a href=
+          "#dom-datachannel-bufferedamount">bufferedAmount</a></code>
+          decreases from above this threshold to equal or below it, the
+          <code title="event-RTCDataChannel-bufferlow"><a href=
+          "#event-datachannel-bufferlow">bufferlow</a></code> event fires.
+          The <code><a href=
+          "#dom-datachannel-lowbufferlimit">lowBufferLimit</a></code>
+          defaults to zero, but the application may change its value at any
+          time.</p>
```
Comment (martinthomson) [inline webrtc.html]: I think that you want to say that this value starts at zero rather than defaults.

Comment (bemasc) [inline webrtc.html]: OK, I've improved the wording here.

**Comment by bemasc** [comment]
I didn't add an `onbufferlow` because I think `onwhatever` is rightly deprecated (and there's no compatibility concern for new events), but if you want to keep parallelism I'm happy to add it.

OK, I've changed the names to `bufferedAmountLowThreshold` and `bufferedamountlow`.

**Comment by jan-ivar** [comment]
@bemasc - Do you have a link about `onwhatever` being deprecated?

**Comment by bemasc** [comment]
[Mozilla](https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Event_handlers) says "If you're working with a modern browser, use addEventListener() instead."

`onwhatever` has been removed entirely from the DOM Level 2 Events and is never mentioned in the [living standard](https://dom.spec.whatwg.org/#events).  Admittedly, I haven't found any explicit guidance on whether to include it for new events or not.

**Comment by jan-ivar** [comment]
Thanks for the link. PeerConnection has `onxxx` for all its events I think, and AFAIK they were all added after this advice.

**Comment by bemasc** [comment]
OK, I've added `onbufferedamountlow`.

**Pull request merged**